### PR TITLE
 Implementation of GQL entity converters

### PR DESCRIPTION
**Pull request by maikmb**
Implementation of usecase (buchu) and entities (gotu) convertion to GQL type based.

- Input
- Type
- Mutation
- Query
- Subscription

### CODE REVIEW

**Comment by maikmb** [comment]
Hi @jhomarolo , can you see my PR?

File: package.json [hunk 1]
```diff
@@ -32,20 +33,23 @@
   "license": "MIT",
   "homepage": "https://github.com/herbsjs/herbs2gql#readme",
   "dependencies": {
-    "gotu": "^1.0.6"
+    "buchu": "^1.0.7",
+    "gotu": "^1.0.6",
+    "lodash": "^4.17.20"
```
Comment (jhomarolo) [inline package.json:38]: Why we need lodash here? Can be inside devdependencies?

**Review by jhomarolo — CHANGES_REQUESTED** [review]
@maikmb I haven't seen enough failure tests on this lib to test its resilience to errors/unexpected scenarios.
Can you include them?

File: package.json (hunk 1 above)
Comment (maikmb) [reply package.json]: Hi @jhomarolo !
I Includes error scenarios for unit testing. Can you see please?

**Comment by maikmb** [comment]
Hi @jhomarolo !
I Includes error scenarios for unit testing. Can you see please?

File: package.json (hunk 1 above)
Comment (maikmb) [reply package.json]: Hi @jhomarolo !
Lodash is used to convert text to `camecasel`. In the future we must implement this native function in **suma**.

**Comment by maikmb** [comment]
image

**Pull request merged**